sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubAsyncClient.java

```diff
@@ -317,25 +317,25 @@ private static String getExpression(EventPosition eventPosition) {
         final String isInclusiveFlag = eventPosition.isInclusive() ? "=" : "";
 
         // order of preference
-        if (eventPosition.offset() != null) {
+        if (eventPosition.getOffset() != null) {
             return String.format(
                 AmqpConstants.AMQP_ANNOTATION_FORMAT, OFFSET_ANNOTATION_NAME.getValue(),
                 isInclusiveFlag,
-                eventPosition.offset());
+                eventPosition.getOffset());
         }
 
-        if (eventPosition.sequenceNumber() != null) {
+        if (eventPosition.getSequenceNumber() != null) {
             return String.format(
                 AmqpConstants.AMQP_ANNOTATION_FORMAT,
                 SEQUENCE_NUMBER_ANNOTATION_NAME.getValue(),
                 isInclusiveFlag,
-                eventPosition.sequenceNumber());
+                eventPosition.getSequenceNumber());
         }
 
-        if (eventPosition.enqueuedDateTime() != null) {
+        if (eventPosition.getEnqueuedDateTime() != null) {
             String ms;
             try {
-                ms = Long.toString(eventPosition.enqueuedDateTime().toEpochMilli());
+                ms = Long.toString(eventPosition.getEnqueuedDateTime().toEpochMilli());
             } catch (ArithmeticException ex) {
                 ms = Long.toString(Long.MAX_VALUE);
```

Comment:
@srnagar `String eventHubName()` is not renamed.

Reply:
Updated